docs/en/seeding.rst

```diff
@@ -75,6 +75,14 @@ The Init Method
 The ``init()`` method is run by Phinx before the run method if it exists. This
 can be used to initialize properties of the Seed class before using run.
 
+The Should Execute Method
+~~~~~~~~~~~~~~~
```

Comment:
Not enough ~ right?

Reply:
It's the same amount as the block above:

```
The Init Method
~~~~~~~~~~~~~~~
```

I can change it though

Reply:
I see what you mean, updated it